controllers/storagecluster/reconcile.go

```diff
@@ -459,9 +465,9 @@ func (r *StorageClusterReconciler) reconcilePhases(
 		// for external cluster, we have a different set of ensure functions
 		// preserve list order
```

Comment:
@aruniiird please review as we are changing the order

Reply:
Order part, we really have to test it out in a real cluster

Reply:
We did a test and it was passed.